Eastern-Research-Group/mywaterway
# Feature/732 grts success stories

## Related Issues:
* [HMW-732](https://jira.epa.gov/browse/HMW-732)

## Main Changes:
* Switched from the example GRTS stories on the Restore tab to the production service.
* Formatted the text returned in the `pollutants` and `poll_sources` response fields (the "Pollutants Addressed" and "Sources of Pollution" display fields).
* Fixed a couple type errors related to incorrect property access.

## Steps To Test:
1. Go to http://localhost:3000/community/dc/restore and open the _Success Stories_ section. Confirm it is populated.
2. Expand some of the stories, and confirm that values in the "Pollutants Addressed" and "Sources of Pollution" fields are separated by commas, not colons.
3. Go to http://localhost:3000/community and click a location on the map without searching for a location. Confirm no property access error is logged to the console.
